src/App.tsx

```diff
@@ -15,47 +17,128 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case SortType.ALPABET:
+
+        return good1.localeCompare(good2);
+      case SortType.LENGTH:
+
+        return good1.length - good2.length;
+      case SortType.NONE:
+
+        return 0;
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
```suggestion
  visibleGoods.sort((good1, good2) => {
    switch (sortType) {
      case SortType.ALPABET:
        return good1.localeCompare(good2);
        
      case SortType.LENGTH:
        return good1.length - good2.length;
        
      default:
        return 0;
    }
  });
```

Code style. And you probably don't need it 'case SortType.NONE'